onepassword/items.go

```diff
@@ -2,10 +2,36 @@ package onepassword
 
 import (
 	"encoding/json"
+	"fmt"
 	"strings"
 	"time"
 )
 
+type ItemState string
+
+const (
+	Active   ItemState = ""
+	Deleted  ItemState = "DELETED"
+	Archived ItemState = "ARCHIVED"
+	Unknown  ItemState = "UNKNOWN"
```

Comment:
I don't think this is part of the Connect API. Can we leave it out?

Reply:
Okay, but it is returned by Connect when something goes wrong.